src/Multi/Search.php

```diff
@@ -54,6 +55,113 @@ public function __construct(Client $client, ?MultiBuilderInterface $builder = nu
         $this->_builder = $builder ?? new MultiBuilder();
         $this->_client = $client;
     }
+    
+    /**
+     * @param string $key
+     * @param mixed  $value
+     *
+     * @return $this
+     */
+    public function setOption($key, $value)
```

Comment:
```suggestion
    public function setOption(string $key, $value): self
```

Argument `$key` can probably be type-hinted